.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.7.7
```

Comment:
docker container uses 3.2 which honestly is ok, as the ci goes back to test 3.0 compatibility anyway